src/main/java/net/datafaker/base/Internet.java

```diff
@@ -152,6 +152,15 @@ public String password(int minimumLength, int maximumLength, boolean includeUppe
         return faker.password().password(config);
     }
 
+    /**
+     * Returns a port number between 1 and 65535
+     *
+     * @return a port number
+     */
+    public int port() {
+        return faker.random().nextInt(1, 65535);
```

Comment:
Based on RFC 6335 https://www.rfc-editor.org/info/rfc6335
allowed range is `[0, 65535]` so it should be `return faker.random().nextInt(0, 65535);`

Reply:
Fixed